tests/test_arrow.py

```diff
@@ -1230,6 +1230,41 @@ def test_arrow_list_feather_columns(self):
 
         os.unlink(f.name)
 
+    def test_arrow_feather_dataset_binary(self):
+        """test_arrow_feather_dataset_binary"""
+        import tensorflow_io.arrow as arrow_io
+
+        from pyarrow import feather as pa_feather
+        import pyarrow as pa
+        import numpy as np
+
+        local_path = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), "test_image", "sample.png"
+        )
+        with open(local_path, "rb") as f:
+            data = [f.read()]
+            table = pa.Table.from_arrays([data], ["data"])
+
+        chunk_size = 1000
+        with tempfile.NamedTemporaryFile(delete=False) as f:
+            pa_feather.write_feather(table, f, chunksize=chunk_size)
+
```

Comment:
@yongtang thanks for adding this feature. I saw that `StringType` is a sub-class of `BinaryType` [Ref](https://arrow.apache.org/docs/cpp/api/datatype.html#_CPPv4N5arrow10StringTypeE) so can we add one more test where we read from a string typed pa table?